scaffold/template/custom/android/build.gradle

```diff
@@ -0,0 +1,38 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext {
+        buildToolsVersion = "29.0.3"
+        minSdkVersion = 21
+        compileSdkVersion = 30
+        targetSdkVersion = 30
```

Comment:
This bumps the target to Android 11 right? Why do we need to change this? 

I'm wondering if there's any potential issues with this change.

Reply:
i dont remember the exact fastlane error right now, but it was asking to upgrade the sdk version. Do you mean we can have errors with previous built apps?